lib/src/write_task.dart

```diff
@@ -9,8 +9,8 @@ enum WriteCommand {
 }
 
 class WriteTask {
-  WriteCommand command;
-  DocumentReference document;
-  Map<String, dynamic> data;
-  bool merge;
+  WriteCommand? command;
+  DocumentReference? document;
+  Map<String, dynamic>? data;
```

Comment:
I think this may be null (e.g., delete) Good.

Reply:
Added a comment to explain when it can be null.